v1/fleet_units/logrotate.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Log Rotator for docker containers
+After=docker.service
+Requires=docker.service
+
+[Service]
+User=core
+Restart=always
+ExecStartPre=-/usr/bin/docker stop logrotate
+ExecStartPre=-/usr/bin/docker rm -f logrotate
+ExecStartPre=/usr/bin/docker pull michaloo/logrotate
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill logrotate
+ExecStartPre=-/usr/bin/docker rm logrotate
+ExecStart=sudo /usr/bin/docker run --name logrotate -v /var/lib/docker:/var/lib/docker michaloo/logrotate
```

Comment:
@iderdik 's comment got lost as this line changed, but I also agree w/ forking and making our own container so that we have full control of the contents of the container.  It's fine for now, but something we will need to do for most containers before Production.